api/tables/store.js

```diff
@@ -26,15 +26,15 @@ export function createStoreTable (region, tableName, options = {}) {
     /**
      * Check if the given link CID is bound to the uploader account
      *
-     * @param {string} uploaderDID
-     * @param {string} payloadCID
+     * @param {import('@ucanto/interface').DID} space
+     * @param {import('../service/types').AnyLink} link
      */
-    exists: async (uploaderDID, payloadCID) => {
+    exists: async (space, link) => {
       const cmd = new GetItemCommand({
         TableName: tableName,
         Key: marshall({
-          uploaderDID,
-          payloadCID,
+          space,
+          link: link.toString()
         }),
         AttributesToGet: ['uploaderDID'],
```

Comment:
space?

Reply:
This is an `exists: (DID, AnyLink) => boolean` function. We don't need any of the items properties. but yes, this is a left over that should change.